### PR TITLE
Update dependabot to update Dockerfile & Github actions too

### PR DESCRIPTION
**Pull request by monotek**
#### What this PR does / why we need it:
- update depnedatbot config to also update Dockerfile and Github actions

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

**Comment by alexzielenski** [comment]
is there any way to see that this works before merging

**Comment by monotek** [comment]
I don´t think so but i use it like this in a lot of repos already.

Example: https://github.com/monotek/fluentd-elasticsearch/blob/master/.github/dependabot.yml

File: .github/dependabot.yml [hunk 1]
```diff
@@ -1,5 +1,15 @@
 version: 2
 updates:
+  - package-ecosystem: "docker"
```
[the remark this answers is not in the document]
Comment (monotek) [reply .github/dependabot.yml:3]: https://github.com/kubernetes-sigs/kubectl-validate/blob/main/Dockerfile

[the remark this answers is not in the document]
Comment (eddycharly) [reply .github/dependabot.yml:3]: Are we using this dockerfile ?
cc @alexzielenski

**Comment by monotek** [comment]
not stale?

**Pull request closed without merging**